packages/core/src/external/fhir/patient/index.ts

```diff
@@ -1,28 +1,25 @@
 import {
-  Identifier,
   Patient as FHIRPatient,
   ContactPoint,
   Reference,
   DocumentReference,
+  Identifier,
 } from "@medplum/fhirtypes";
 import { driversLicenseURIs, identifierSytemByType } from "../../../domain/oid";
 import { ContactTypes, Contact } from "../../../domain/contact";
 import { Address } from "../../../domain/address";
-import { GenderAtBirth, Patient, PersonalIdentifier, splitName } from "../../../domain/patient";
+import { Patient, splitName, genderAtBirthMapping } from "../../../domain/patient";
 import { getIdFromSubjectId, getIdFromSubjectRef } from "../shared";
 
-export const genderMapping: { [k in GenderAtBirth]: "female" | "male" } = {
-  F: "female",
-  M: "male",
-};
+export type PatientIdAndData = Pick<Patient, "id" | "data">;
+// TODO Better name
+export type StrongId = Omit<Identifier, "value" | "system"> & { value: string; system: string };
```

Comment:
What are your thoughts here? My thinking is that `StrongId` is limited to CW, this is a version of FHIR's `Identifier` w/ required fields - maybe we can call it as such, like `IdentifierWithValueAndSystem`? :)

Reply:
By the way, does it need to have those fields as required? It seems the only place that uses the response of `getFhirStrongIdsFromPatient` is that utils script, which returns a FHIR `Patient`, so not sure why we need this type.

Reply:
Agreed -- the return type is `Patient` is from medplum, which means any "receiving entity" from the outcome of the main function should just assume it's an `Identifier` and should be forced to check system / value if they want to use them explicitly. The migration uses this function out of turn, but is doing FHIR so I also agree with you there.